src/translations/main_pl.ts

```diff
@@ -465,19 +465,19 @@ OpenSSL może nie być zainstalowane.</translation>
     </message>
     <message>
         <source>Start Thunderbird by clicking on the tray icon if it was closed</source>
-        <translation type="unfinished"></translation>
+        <translation>Uruchom Thunderbird klikając w ikonę na tecce systemowej jeśli został zamknięty.</translation>
     </message>
     <message>
         <source>When detecting a change, wait</source>
-        <translation type="unfinished"></translation>
+        <translation>Jeśli wykryto zmianę, poczekaj</translation>
     </message>
     <message>
         <source> milliseconds</source>
-        <translation type="unfinished"></translation>
+        <translation>milisekund</translation>
```

Comment:
```suggestion
        <translation> milisekund</translation>
```
There will be a number in front of this, let's add a space like we do for ` second` → ` sekund`.

Reply:
I see three more :

        <source>At startup</source>
        <translation">Przy uruchamianiu</translation>
   
        <source>When hiding Thunderbird</source>
        <translation>Podczas ukrywania Thunderbird</translation>
 
        <source>When showing Thunderbird</source>
        <translation>Podczas wyświetlania Thunderbird</translation>

I upgraded file.